src/python/WMCore/WMBS/MySQL/Files/AddRunLumi.py

```diff
@@ -36,7 +36,8 @@ def getBinds(self, filename=None, runs=None):
                 for lumi in run:
```

Comment:
Here you could get eventsPerLumi (or whatever it is) from Run and iterate over the key/value pairs. Keys are lumis, events are values. It would avoid a ton of function calls. 

```
for lumi, events in run.eventsPerLumi:
```